R/athena_connection.R

```diff
@@ -42,6 +45,7 @@ connect_athena <- function(aws_region = NULL,
                            rstudio_conn_tab = FALSE,
                            session_duration = 3600,
                            role_session_name = NULL,
+                           schema_name = "default",
```

Comment:
@pjrh-moj, shouldn't this be `schema_name = "__temp__"`?

Reply:
I didn't want to change the default behaviour - so this pulls through the `schema_name` argument from Noctua which was always set as "default" behind the scenes (unless overridden by a named argument passed by `...`).

Also, if we set `schema_name = "__temp__"` then everyone will have a temp db created for them on connection, which slows things down for those who don't need it and a whole bunch of dbs are created which aren't needed.

This PR adds the functionality to use the temp db as the schema for a connection, but doesn't make it the default. Does that make sense?

Reply:
Yep, thank you.